core/src/main/java/com/netease/arctic/catalog/CatalogLoader.java

```diff
@@ -50,13 +51,15 @@
 /** Catalogs, create catalog from arctic metastore thrift url. */
 public class CatalogLoader {
 
-  public static final String AMS_CATALOG_IMPL = BasicArcticCatalog.class.getName();
   public static final String ICEBERG_CATALOG_IMPL = BasicIcebergCatalog.class.getName();
   public static final String HIVE_CATALOG_IMPL =
       "com.netease.arctic.hive.catalog.ArcticHiveCatalog";
   public static final String GLUE_CATALOG_IMPL = "org.apache.iceberg.aws.glue.GlueCatalog";
```

Comment:
We can use `GlueCatalog.class.getName()` now, since `iceberg-aws` has been included to the dependencies.